api/v1alpha3/azurecluster_conversion.go

```diff
@@ -85,6 +96,19 @@ func (dst *AzureCluster) ConvertFrom(srcRaw conversion.Hub) error { // nolint
 		return err
```

Comment:
Looks like there are 2 calls to utilconversion.MashalData. There should only be one.